gateway/src/apicast/policy.lua

```diff
@@ -18,7 +18,8 @@ local PHASES = {
     'ssl_certificate',
 }
 
-local setmetatable = setmetatable
+local GC = require('apicast.gc')
+local setmetatable = GC.setmetatable
```

Comment:
Maybe we should rename this var to avoid confusion.
When I read this diff for the first time I assumed that we were using the normal `setmetatable` function.

Reply:
Yep.